.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
 
 install:
   - ./pkg-build.sh install_deps
-  - ./pkg-build.sh install_github jimhester/covr
+  - ./pkg-build.sh install_github jimhester/covr hadley/testthat#443
```

Comment:
I'm not in a hurry to have this merged, unless you plan a major refactoring spree.